### DISCUSSION TITLE
Is it necessary for Linkerd control-plane services to be Meshed

### DISCUSSION THREAD

**Discussion by mbastawisy-careem** [linkerd/linkerd2#10880] [Q&A] [answered]
Currently, we have Linkerd deployed on our clusters, but we’ve noticed that the control plane itself is meshed. We’re curious if this is the correct setup and if there might be any potential ‘chicken-and-egg’ scenarios due to this configuration. We didn’t customize the default options for the helm chart, so we are using the default one.

**Comment by alpeb** [comment, marked as answer]
Hi, yes, the control plane requires to be injected, so that's the correct setup, and there's no special config required out of the box.
